metabase/metabase
# Navbar Improvements: Updated UI

### Description

Navbar UI work as part of https://github.com/metabase/metabase/issues/36370. This does not fully implement any of the new heuristics for the navbar logic, but aims to accomplish the visuals needed as part of the updates.

Note this doesn't implement hover states as I'm waiting on some feedback from design. Will follow up on those.

### How to verify

- View any page that has the navbar
- Navbar button should open / close in the same way it does today
- Logo should take you to home page now
- Check mobile / desktop

### Demo
<img width="1492" alt="Screenshot 2024-01-17 at 2 22 47 PM" src="https://github.com/metabase/metabase/assets/7104357/28bec3fd-dfd2-4d0a-8b1e-6e9c9e7706ea">
<img width="1491" alt="Screenshot 2024-01-17 at 2 23 07 PM" src="https://github.com/metabase/metabase/assets/7104357/b2657738-ef47-4746-9bc4-9c1e06843bff">
<img width="1490" alt="Screenshot 2024-01-17 at 2 22 56 PM" src="https://github.com/metabase/metabase/assets/7104357/3a40a20a-68fb-45a4-badb-7d3ba14170c9">


### Checklist
